src/main/java/nl/nn/testtool/metadata/StatusMetadataFieldExtractor.java

```diff
@@ -18,28 +18,59 @@
 import java.util.List;
 
 import nl.nn.testtool.Checkpoint;
+import nl.nn.testtool.MetadataFieldExtractor;
 import nl.nn.testtool.Report;
 
 /**
  * @author Jaco de Groot
  */
 public class StatusMetadataFieldExtractor extends DefaultValueMetadataFieldExtractor {
-	
+	private MetadataFieldExtractor delegate = null;
+	private String otherLabelForError = null;
```

Comment:
Can you change this to:
private String errorStatus = "Error";
So that we have Error as default text and you can override it by calling setErrorStatus()?
And maybe also add:
private String successStatus = "Succes";
And the corresponding setSuccessStatus()

Reply:
Gedaan.